sys/androidmedia/gstamc.c

```diff
@@ -653,6 +678,159 @@ gst_amc_codec_get_release_method_id (GstAmcCodec * codec)
   return media_codec.release_output_buffer;
 }
 
+
```

Comment:
The method id should be cached as any other method already cached in get_java_classes

Reply:
catching done